### PR TITLE
Add Heidelberg

### PR DESCRIPTION
**Pull request by katrinleinweber**
LGTM 😁 

Screenshot 2022-05-25 at 08 56 07

### CODE REVIEW

**Review by mojoaxel — APPROVED** [review]
LGTM

File: cities/heidelberg.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                "title": "Wochenmarkt Rohrbach",
+                "location": "Rathausplatz, 69126 Heidelberg",
+                "opening_hours": "Sa 08:00-13:00"
+            },
+            "type": "Feature"
+        }
+    ],
+    "metadata": {
+        "data_source": {
+            "title": "Stadt Heidelberg",
```
[the remark this answers is not in the document]
Comment (katrinleinweber) [reply cities/heidelberg.json]: Sure thing :-) I checked how the other cities do it with `rg '\b20\d{2}\b' cities/*.json | rg title` ([using `ripgrep` there](https://github.com/BurntSushi/ripgrep/blob/master/GUIDE.md)) and went with the majority.

**Review by johnjohndoe — APPROVED** [review]
@katrinleinweber Thanks a lot for adding yet another city (and the latest commit update)! :+1: 

I just fixed a typo in the README.md and simplified an opening hours entry. Other than that everything was perfect :heart_eyes_cat: 

As before, I invite you to contact the city of Heidelberg to put this website on their website to inform citizens about it. Also, a social media post with screenshot helps a lot. See issue #38, too.

**Comment by johnjohndoe** [comment]
@katrinleinweber, here are a few advertisement ideas - as you already know this website is **nothing without its users**.

- [ ] Tweet it out - don't forget a nice screenshot :bird: 
- [ ] Ask the city of Heidelberg if they put a link and maybe a screenshot on their website. Some cities did that and one can clearly see the user increase in the stats! :cityscape: 
- [ ] Share the news with local newspapers :newspaper: 
- [ ] Print stickers and share them: the assets can be found in the [wo-ist-markt-media repository](https://github.com/wo-ist-markt/wo-ist-markt-media) :printer: 
- [ ] Become a living advertisement - print your own t-shirt - I did that! Same assets as above :shirt: 

Of course, if you come up with something ever cooler - please let me know!

**Pull request merged**